lib/browser/lighthouse.js

```diff
@@ -163,9 +159,50 @@ exports.assertLighthouseScore = function assertLighthouseScore(
     assert.expect(parseLhResult(res).isSuccess(score));
   };
 
-  return this.getLighthouseData(flags, config || lhConfig).then(assertScore);
+  return this.getLighthouseData(flags, getAccessibilityConfig(config)).then(
+    assertScore
+  );
 };
 
-exports.runLighthouseAudit = function runLighthouseAudit(flags, config) {
-  return this.getLighthouseData(flags, config || lhConfig).then(parseLhResult);
+/**
+ *
+ * @param {string|Array<string>} categories - LH categories: 'performance', 'accessibility', 'best-practices', 'seo', 'pwa'
+ * @param {Object=} lhConfig - Custom Lighthouse config
+ */
+exports.runLighthouse = function runLighthouse(categories = [], lhConfig = {}) {
+  categories = !Array.isArray(categories) ? [categories] : categories;
+  let config = Object.assign({}, defaultConfig);
```

Comment:
similar problem as above; you could either do a deep clone here, or in all of the merges below, do them like: `config = merge({}, config, catConfig)`